packages/xsnap/src/xsnap.js

```diff
@@ -146,7 +198,17 @@ export async function xsnap(options) {
     console.log('XSNAP_DEBUG_RR', { bin, args });
   }
   const xsnapProcess = spawn(bin, args, {
-    stdio: ['ignore', stdout, stderr, 'pipe', 'pipe'],
+    stdio: [
+      'ignore', // 0: stdin
+      stdout, // 1: stdout
+      stderr, // 2: stderr
+      'pipe', // 3: messagesToXsnap
+      'pipe', // 4: messagesFromXsnap
+      'ignore', // 5: XSBug
+      'ignore', // 6: XSProfiler
+      snapshotUseFs ? 'ignore' : 'pipe', // 7: savedSnapshotsStream
+      snapshotUseFs || !snapshotStream ? 'ignore' : 'pipe', // 8: loadSnapshotsStream
```

Comment:
this one can only be used for a single snapshot, right? so maybe singular `loadSnapshotStream` or `snapshotLoadStream`

Reply:
Technically we can re-use it, but yeah we only use it once.

Reply:
"technically" as in we don't close it, right? But the only way to make xsnap read from it is with a CLI argument (which can only be provided once), vs the writing side which is triggered by a command-pipe message (which can happen multiple times).

Reply:
No we could introduce a new command that says, toss away your VM and create a new one from this stream (or file). There is no reason snapshot load has to be a command line thing. However I agree the name would be clearer and we can rename if we ever change how snapshots are loaded.